src/app/utils/encrypt-submission.ts

```diff
@@ -0,0 +1,56 @@
+import { Request } from 'express'
+
+import { IPopulatedForm } from 'src/types'
+
+import { createLoggerWithLabel } from '../../config/logger'
+import { getProcessedResponses } from '../modules/submission/submission.service'
+
+import { checkIsEncryptedEncoding } from './encryption'
+import { createReqMeta } from './request'
+
+const logger = createLoggerWithLabel(module)
+
+type ReqWithForm<T> = T & { form: IPopulatedForm }
+
+export const isValidEncryptSubmission = (req: Request): boolean => {
```

Comment:
Utility functions should also not need to be aware of other layers, in this case this function needs to know about `req` from the controller layer, which is not what we want